rts/Lua/LuaOpenGL.cpp

```diff
@@ -3627,12 +3628,8 @@ int LuaOpenGL::GenerateMipmap(lua_State* L)
 	if (tex == nullptr)
 		return 0;
 
-	GLint currentBinding;
-	assert(LuaTextures::Format2Query.find(tex->target) != LuaTextures::Format2Query.end());
-	glGetIntegerv(LuaTextures::Format2Query.find(tex->target)->second, &currentBinding);
-	glBindTexture(tex->target, tex->id);
+	auto texBind = GL::TexBind(tex->target, tex->id);
```

Comment:
Let's rename the class into `ScopedTexBind`

Reply:
I don't mind. I tried to use shortest understandable version, because this is kind of a spam object.

Reply:
> Let's rename the class into `ScopedTexBind`

I am not sure if adding "Scoped" would serve well though.

Reply:
> Let's rename the class into `ScopedTexBind`

The problem with "ScopedTexBind" is that when we write
auto texBind = ScopedTexBind(...)
First of all, it is already obvious this is scoped, because we are creating it as an object.
I am before many things trying to achieve the best readability, and when you read "Scoped...", you read exactly "Scoped what"? Scoped tex bind, ah yes.
This is what I mean when I say "Scoped" may not serve well.

Reply:
I propose to just add a short description on what the object does, in TexBind.h, so a new user doesn't mistake it for else, rather than expanding its name.